arrows/mvg/applets/CMakeLists.txt

```diff
@@ -18,6 +18,11 @@ set( headers
 # Add our plugin path to the default list
 include_directories( ${CMAKE_CURRENT_BINARY_DIR} )
 
+if (KWIVER_ENABLE_TESTS)
+  add_subdirectory(tests)
+  list(APPEND kwiver_plugin_applets_subdir tests)
```

Comment:
What does this line do?

Reply:
That line was a holdover from a previous attempt at writing the tests, I've now removed it.